src/modules/logger/params.c

```diff
@@ -59,13 +59,35 @@ PARAM_DEFINE_INT32(SDLOG_UTC_OFFSET, 0);
  * @value 1 from boot until disarm
  * @value 2 from boot until shutdown
  *
- * @min 0
- * @max 2
  * @reboot_required true
  * @group SD Logging
  */
 PARAM_DEFINE_INT32(SDLOG_MODE, 0);
 
+/**
+ * Mission Log
+ *
+ * If enabled, a second mission log file will be written to the SD card.
```

Comment:
HOw about:

```
/**
 * Mission Log
 *
 * If enabled, a small additional "mission" log file will be written to the SD card.
 * The log is intended for tasks like generating flight statistics and geotagging, and 
 * a lot fewer messages than the full log.
 *
 * The different modes can be used to further reduce the logged data
 * (and thus the log file size). For example, choose geotagging mode to
 * only log data required for geotagging.

 * Note that the normal/full log is still created, and contains all
 * the data in the mission log (and more).
 */
```

The `@value 1 Complete` was also a little confusing to me when I first saw it, because I thought maybe this meant "full log", rather than just "Complete mission log".
Maybe it is ok. Alternative: 
```
 * @value 0 Disabled
 * @value 1 All mission messages
 * @value 2 Geotagging mission messages
```

Reply:
Thanks, updated accordingly.